api/src/main/java/com/epam/pipeline/manager/cloud/azure/AbstractAzurePriceListLoader.java

```diff
@@ -0,0 +1,252 @@
+/*
```

Comment:
It seems to me that child classes of `AbstractAzurePriceListLoader` have almost identical code. The main difference is in the method `getPricing`. Is it possible to load prices from EA and RateCard in one format (e.g. make `AzureEAPricingMeter` and `AzurePricingMeter` to implement some common interface with all required methods `getMeterId`, etc.)? In this case we can have a small loader classes for EA and RateCard but keep all further processing in common code.